acdh-oeaw/vicav-app
# Add optional config.json to configure map zoom and center

Feature:
* A config file config.json is loaded from the app root (served by http.xqm), which is used for getting the map zoom and center properties. This is a simple UX improvement for Tunocent team which enables them to focus right away on one region.

Tests: 
* Ensure that app is functional without the config file.
* Ensure app is functional with the config file and zoomed up (not testing explicitly for center though).
Later we can use this to add more local configuration options to different VICAV instances like home page windows.
